packages/nextra-theme-blog/tsup.config.ts

```diff
@@ -7,6 +7,6 @@ export default defineConfig({
   dts: true,
   name: 'nextra-theme-blog',
   outExtension: () => ({ js: '.js' }),
-  target: tsconfig.compilerOptions.target,
+  target: tsconfig.compilerOptions.target as 'es2016',
```

Comment:
I guess there is no workaround to avoid using `as` and take `const` value of target? found active issue https://github.com/microsoft/TypeScript/issues/32063

Reply:
I'm not sure what's better way to do it